content/update-checker/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,101 @@
+= Axon Framework Update Checker
+
+The update checker is a new feature included in the upcoming Axon Framework 5, which ensures the security of the Axon Framework application and its modules and provides useful information to its maintainers.
+
+== What does the update checker do?
+
+**1. Collect anonymous data** The update checker automatically and anonymously sends technical information about your Axon Framework installation to AxonIQ.
```

Comment:
Shouldn't we inverse 1 and 2. 
What does the update checker do? -> It checks for updates
What information does the checker send? -> It sends version information, machine id, etc.
How do you use that information -> We check versions against the latest and do some (anonymous) processing of the data to capture usage patterns.
Why do we do this (this text is already there) -> because we want to understand our community better

Reply:
Inversed. Let me know what you think